app/assets/javascripts/blacklight/facet_load.js

```diff
@@ -0,0 +1,14 @@
+//= require blacklight/core
```

Comment:
If `blacklight/core` is already being required, do we need to require it here again?